src/java/exception/MaxPassengerExceededException.java

```diff
@@ -9,6 +9,7 @@
     public MaxPassengerExceededException(int maxPassengers, int expectedPassengersLoad) {
         this.maxPassengers = maxPassengers;
         this.expectedPassengersLoad = expectedPassengersLoad;
+        System.out.println("Za dużo pasażerów! Maksymalna ilość miejsc wynosi "+ getMaxPassengers());
```

Comment:
That's very bad idea. Exception class is just to keep exception data, not to handle exception (by printing it you're kind'of handling them). This should be in catch block